docs/standards/smirnoff.md

```diff
@@ -521,14 +528,48 @@ For example:
 
 Currently, only `potential="k*(1+cos(periodicity*theta-phase))"` is supported, where we utilize the functional form:
 ```
-U = \sum_{i=1}^N k_i * (1 + cos(periodicity_i * phi - phase_i))
+U = \sum_{i=1}^N k_i/idivf_i * (1 + cos(periodicity_i * phi - phase_i))
 ```
 
 !!! note
     **AMBER defines a modified functional form**, such that `U = \sum_{i=1}^N (k_i/2) * (1 + cos(periodicity_i * phi - phase_i))`, so that barrier heights would need to be divided by two in order to be used in the SMIRNOFF format.
```

Comment:
(not blocking)
```suggestion
    **AMBER defines a modified functional form**, such that `U = \sum_{i=1}^N (k_i/2) * (1 + cos(periodicity_i * phi - phase_i))`, so that barrier heights would need to be divided by two in order to be used in the SMIRNOFF format (AMBER uses phi in an analogous way to how SMIRNOFF uses theta).
```